packages/core/snapshot/manager.go

```diff
@@ -0,0 +1,175 @@
+package snapshot
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/iotaledger/goshimmer/packages/core/epoch"
+	"github.com/iotaledger/goshimmer/packages/core/ledger"
+	"github.com/iotaledger/goshimmer/packages/core/notarization"
+	"github.com/iotaledger/goshimmer/packages/core/tangleold"
+	"github.com/iotaledger/goshimmer/packages/node/database"
+	"github.com/iotaledger/hive.go/core/generics/event"
+	"github.com/iotaledger/hive.go/core/kvstore"
+)
+
+const (
+	prefixSolidEntryPoint byte = iota
+)
+
+// Manager is the snapshot manager.
+type Manager struct {
+	tangle          *tangleold.Tangle
```

Comment:
We only use the tangle for its `CommittmentFunc`, don't we have it already through the notarizationMgr dependency?